### PR TITLE
Disable ndsu security contacts update

### PR DESCRIPTION
**Pull request by mmascher**

### CODE REVIEW

File: topology/Donald Danforth Plant Science Center/OSG_US_DANFORTH_ce04/OSG_US_DANFORTH_HOSTED_CE.yaml [hunk 1]
```diff
@@ -3,16 +3,16 @@ GroupID: 471
 Production: true
 Resources:
   OSG_US_DANFORTH_ce04:
-    Active: true
+    Active: false 
     ContactLists:
       Administrative Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/Donald Danforth Plant Science Center/OSG_US_DANFORTH_ce04/OSG_US_DANFORTH_HOSTED_CE.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/Donald Danforth Plant Science Center/OSG_US_DANFORTH_ce04/OSG_US_DANFORTH_HOSTED_CE.yaml [hunk 2]
```diff
@@ -3,16 +3,16 @@ GroupID: 471
...
-          Name: Suchandra Thapa
+          Name: Marco Mascheroni
       Security Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/Donald Danforth Plant Science Center/OSG_US_DANFORTH_ce04/OSG_US_DANFORTH_HOSTED_CE.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/Florida State University/FSU_HNPGRID/OSG_US_FSU_HNPGRID.yaml [hunk 3]
```diff
@@ -8,15 +8,15 @@ Resources:
       Administrative Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/Florida State University/FSU_HNPGRID/OSG_US_FSU_HNPGRID.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/Florida State University/FSU_HNPGRID/OSG_US_FSU_HNPGRID.yaml [hunk 4]
```diff
@@ -8,15 +8,15 @@ Resources:
...
-          Name: Suchandra Thapa
+          Name: Marco Mascheroni
       Resource Report Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/Florida State University/FSU_HNPGRID/OSG_US_FSU_HNPGRID.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/Florida State University/FSU_HNPGRID/OSG_US_FSU_HNPGRID.yaml [hunk 5]
```diff
@@ -8,15 +8,15 @@ Resources:
...
-          Name: Suchandra Thapa
+          Name: Marco Mascheroni
       Security Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/Florida State University/FSU_HNPGRID/OSG_US_FSU_HNPGRID.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/Georgia State University/OSG_US_GSU_ACORE/OSG_US_GSU_ACORE.yaml [hunk 6]
```diff
@@ -8,11 +8,11 @@ Resources:
       Administrative Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/Georgia State University/OSG_US_GSU_ACORE/OSG_US_GSU_ACORE.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/Georgia State University/OSG_US_GSU_ACORE/OSG_US_GSU_ACORE.yaml [hunk 7]
```diff
@@ -8,11 +8,11 @@ Resources:
...
-          Name: Suchandra Thapa
+          Name: Marco Mascheroni
       Security Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/Georgia State University/OSG_US_GSU_ACORE/OSG_US_GSU_ACORE.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/Inter-University Centre for Astronomy and Astrophysics/IUCAA Sarathi Cluster/OSG_IN_IUCAA_SARATHI.yaml [hunk 8]
```diff
@@ -8,11 +8,11 @@ Resources:
       Administrative Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/Inter-University Centre for Astronomy and Astrophysics/IUCAA Sarathi Cluster/OSG_IN_IUCAA_SARATHI.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/Inter-University Centre for Astronomy and Astrophysics/IUCAA Sarathi Cluster/OSG_IN_IUCAA_SARATHI.yaml [hunk 9]
```diff
@@ -8,11 +8,11 @@ Resources:
...
-          Name: Suchandra Thapa
+          Name: Marco Mascheroni
       Security Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/Inter-University Centre for Astronomy and Astrophysics/IUCAA Sarathi Cluster/OSG_IN_IUCAA_SARATHI.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/North Dakota State University/OSG_US_NDSU_CCAST_CLUSTER2/OSG_US_NDSU_CCAST_CLUSTER2.yaml [hunk 10]
```diff
@@ -3,16 +3,16 @@ GroupID: 480
 Production: true
 Resources:
   OSG_US_NDSU_CCAST_CLUSTER2:
-    Active: true
+    Active: false
     ContactLists:
       Administrative Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/North Dakota State University/OSG_US_NDSU_CCAST_CLUSTER2/OSG_US_NDSU_CCAST_CLUSTER2.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/North Dakota State University/OSG_US_NDSU_CCAST_CLUSTER2/OSG_US_NDSU_CCAST_CLUSTER2.yaml [hunk 11]
```diff
@@ -3,16 +3,16 @@ GroupID: 480
...
-          Name: Suchandra Thapa
+          Name: Marco Mascheroni
       Security Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/North Dakota State University/OSG_US_NDSU_CCAST_CLUSTER2/OSG_US_NDSU_CCAST_CLUSTER2.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/North Dakota State University/OSG_US_NDSU_CCAST_CLUSTER2/OSG_US_NDSU_CCAST_CLUSTER2.yaml [hunk 12]
```diff
@@ -27,11 +27,11 @@ Resources:
       Administrative Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/North Dakota State University/OSG_US_NDSU_CCAST_CLUSTER2/OSG_US_NDSU_CCAST_CLUSTER2.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/North Dakota State University/OSG_US_NDSU_CCAST_CLUSTER2/OSG_US_NDSU_CCAST_CLUSTER2.yaml [hunk 13]
```diff
@@ -27,11 +27,11 @@ Resources:
...
-          Name: Suchandra Thapa
+          Name: Marco Mascheroni
       Security Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/North Dakota State University/OSG_US_NDSU_CCAST_CLUSTER2/OSG_US_NDSU_CCAST_CLUSTER2.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/University of Utah/CHPC/CHPC Group.yaml [hunk 14]
```diff
@@ -8,15 +8,15 @@ Resources:
       Administrative Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/University of Utah/CHPC/CHPC Group.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/University of Utah/CHPC/CHPC Group.yaml [hunk 15]
```diff
@@ -8,15 +8,15 @@ Resources:
...
-          Name: Suchandra Thapa
+          Name: Marco Mascheroni
       Resource Report Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/University of Utah/CHPC/CHPC Group.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/University of Utah/CHPC/CHPC Group.yaml [hunk 16]
```diff
@@ -8,15 +8,15 @@ Resources:
...
-          Name: Suchandra Thapa
+          Name: Marco Mascheroni
       Security Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/University of Utah/CHPC/CHPC Group.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/University of Utah/CHPC/CHPC Group.yaml [hunk 17]
```diff
@@ -31,15 +31,15 @@ Resources:
       Administrative Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/University of Utah/CHPC/CHPC Group.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/University of Utah/CHPC/CHPC Group.yaml [hunk 18]
```diff
@@ -31,15 +31,15 @@ Resources:
...
-          Name: Suchandra Thapa
+          Name: Marco Mascheroni
       Resource Report Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/University of Utah/CHPC/CHPC Group.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/University of Utah/CHPC/CHPC Group.yaml [hunk 19]
```diff
@@ -31,15 +31,15 @@ Resources:
...
-          Name: Suchandra Thapa
+          Name: Marco Mascheroni
       Security Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/University of Utah/CHPC/CHPC Group.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/University of Utah/CHPC/CHPC Group.yaml [hunk 20]
```diff
@@ -54,7 +54,7 @@ Resources:
       Resource Report Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/University of Utah/CHPC/CHPC Group.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/University of Utah/CHPC/CHPC Group.yaml [hunk 21]
```diff
@@ -63,6 +63,21 @@ Resources:
         Description: Compute Element
         Details:
           hidden: false
+  OSG_US_UUTAH_NOTCHPEAK:
+    Active: true
+    ContactLists:
+      Resource Report Contact:
+        Primary:
+          ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/University of Utah/CHPC/CHPC Group.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/Wayne State University/OSG_US_WSU_GRID_ce2/OSG_US_WSU_GRID.yaml [hunk 22]
```diff
@@ -8,11 +8,11 @@ Resources:
       Administrative Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/Wayne State University/OSG_US_WSU_GRID_ce2/OSG_US_WSU_GRID.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

File: topology/Wayne State University/OSG_US_WSU_GRID_ce2/OSG_US_WSU_GRID.yaml [hunk 23]
```diff
@@ -8,11 +8,11 @@ Resources:
...
-          Name: Suchandra Thapa
+          Name: Marco Mascheroni
       Security Contact:
         Primary:
           ID: bd2e3964b9b7531c09c58fda7702794119e3fc6e
```
Comment (matyasselmeci) [inline topology/Wayne State University/OSG_US_WSU_GRID_ce2/OSG_US_WSU_GRID.yaml]:
```suggestion
          ID:  030408ab932e143859b5f97a2d1c9e30ba2a9f0d
```

**Comment by brianhlin** [comment]
Ping @mmascher

**Comment by mmascher** [comment]
Changes "batch committed"

**Pull request merged**